charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -342,6 +342,9 @@ controller:
   # Enable dnsConfig for the controller and node pods
   dnsConfig: {}
 node:
+  # Enable SELinux-only optimizations on the EBS CSI Driver node pods
+  # Must only be set true if all linux nodes in the DaemonSet have SELinux enabled
```

Comment:
Non-blocking comment: Do we want to recommend  [additional daemonsets feature](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/d99da6d023b7523a7998c0b1e21a8f741e21349a/docs/additional-daemonsets.md?plain=1) in-line here. 

An extra sentence like: "Please rely on additional daemonsets feature if this is not the case."

Reply:
That's an advanced feature the user should think about before enabling so I'd honestly rather not give an affirmative suggestion for this relatively minor feature.